.github/ISSUE_TEMPLATE/release_tracking.md

```diff
@@ -0,0 +1,77 @@
+---
+name: 🚋 Release cycle tracking
+about: Create a new release cycle tracking issue for a minor release
+title: Tasks for v<release-tag> release cycle
+labels: ''
+assignees: ''
+
+---
+
+Please see the corresponding section in [release-tasks.md](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md) for documentation of individual tasks.  
+
+## Tasks
+
+**Notes**:
+* Weeks are only specified to give some orientation.
+* The following is based on the v1.4 release cycle. Modify according to the tracked release cycle.
+
+Week -3 to 1:
+* [ ] [Release Lead] Set a tentative release date for the minor release
+* [ ] [Release Lead] Assemble release team
+
+Week 1:
+* [ ] [Release Lead] Finalize release schedule and team
+* [ ] [Release Lead] Prepare main branch for development of the new release
+* [ ] [Release Lead] Create a new GitHub milestone
+* [ ] [Communications Manager] Add docs to collect release notes for users and migration notes for provider implementers
+* [ ] [Communications Manager] Update supported versions
+
+Week 1 to 4:
+* [ ] [Release Lead] [Track] Remove previously deprecated code
+
+Week 6:
+* [ ] [Release Lead] Cut the v1.3.1 release
+
+Week 9:
+* [ ] [Release Lead] Cut the v1.3.2 release
+
+Week 11 to 12:
+* [ ] [Release Lead] [Track] Bump dependencies
+
+Week 13:
+* [ ] [Release Lead] Cut the v1.4.0-beta.0 release
```

Comment:
will we have to update the template later when we are past v1.4.0? There are a lot of points in the list that have a specific version.

Reply:
My idea was:

> * The following is based on the v1.4 release cycle. Modify according to the tracked release cycle.

Essentially I would keep the template the same, but the release lead has to customize the release versions and also the weeks (based on the release timeline), ... when they create the issue for a specific release